server/src/main/java/io/littlehorse/server/streams/taskqueue/PollTaskRequestObserver.java

```diff
@@ -51,7 +55,8 @@ public void onError(Throwable t) {
                 taskQueueManager.backend.getInstanceId(),
                 clientId,
                 taskDefId);
-        taskQueueManager.onRequestDisconnected(this);
+        taskQueueManager.onRequestDisconnected(
+                this, new TenantIdModel(requestContext.authorization().tenantId()));
```

Comment:
Seems like the `PollTaskRequestObserver` should have a `getTenantId()`, rather than passing it around.